public/index.html

```diff
@@ -57,6 +57,25 @@
   <div id="app"></div>
   <!-- built files will be auto injected -->
 
+  <script>
+    // check for the current version on the server and refresh the page if that is the local version is outdated
+    // in plain JS so this is served with the index.html ans executed immediatly
+    if (process.env.NODE_ENV === "production") {
```

Comment:
```suggestion
    if ("<%= process.env.NODE_ENV %>" === "production") {
```

Reply:
@Bitcoinera this should fix it. Good find!

Reply:
I used the template syntax in line 64 but forgot it here

Reply:
Great! I thought the fix would be much harder. Tomorrow I'll retest this 😉